scripts/run-builder.sh

```diff
@@ -19,6 +19,9 @@ npm run create-build-status
 BUILDER_RUN=$(tail -1 builder-run.txt)
 npm run install-projects >project-install-build-logs.txt 2>project-install-build-error.txt || npm run record-build-error -- --service=project-install
 test -s project-install-build-error.txt && npm run record-build-error -- --service=project-install
+cd packages/projects
```

Comment:
localProjectDirectory in the script that has been disabled points to packages/projects/projects, which im a bit confused about how it achieves `npm i` successfully for all the projects anyway, but wondering why this is not the same